src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_ForEachStatement.cs

```diff
@@ -57,11 +57,11 @@ public override BoundNode VisitForEachStatement(BoundForEachStatement node)
 
         /// <summary>
         /// Lower a foreach loop that will enumerate a collection using an enumerator.
-        /// 
+        ///
         /// E e = ((C)(x)).GetEnumerator()
         /// try {
         ///     while (e.MoveNext()) {
-        ///         V v = (V)(T)e.Current;
+        ///         V v = (V)(T)e.Current;  -OR-  (D1 d1, ...) = (V)(T)e.Current;
```

Comment:
I believe the `(V)` cast represents the type in `foreach (V v in x)` from the spec. In the deconstruct case, is there a `V`?